### ISSUE TITLE
OpenSubtitles.org API is deprecated

### ISSUE THREAD

**Issue by shalak** [QNapi/qnapi#196]
The OpenSubtitles.org API has [been disabled as of 2023-12-31 for non-VIP members](
https://forum.opensubtitles.org/viewtopic.php?t=17930#p47873).

Are there any plans to migrate to [OpenSubtitles.com REST API](https://opensubtitles.stoplight.io/docs/opensubtitles-api/e3750fd63a100-getting-started) that replaced the old one?

**Comment by semeion** [comment]
It could be nice.

**Comment by Infiexe** [comment]
Yeah QNapi stopped downloading proper subs from OpenSubtitles just now
